README.md

```diff
@@ -12,6 +12,14 @@ examples, and of course your own application code.
 Migrating away from Formik appears to be a necessary eventuality. React Hook Form is an excellent alternative with
 similar features and patterns, making migration less painful than you might think.
 
+The following badge will show if there are known security vulnerabilities in the Formik library, which is a **strong** 
+signal that your migration should be a priority.
+
+[![Known Formik Vulnerabilities](https://snyk.io/test/github/{username}/{repo}/badge.svg)](https://snyk.io/test/github/jaredpalmer/formik)
```

Comment:
I don't see the vulnerabilities on this page...would that require a Snyk login?

Reply:
You should basically see something like 
<img width="1048" alt="image" src="https://github.com/trussworks/formik-to-react-hook-form-migration-guide/assets/15805554/38d19155-59a1-4a41-8bb5-1b6afcb2c308">

The link _should_ render as a badge when merged... I hope, kinda like this:
<img width="173" alt="image" src="https://github.com/trussworks/formik-to-react-hook-form-migration-guide/assets/15805554/b796c012-b4eb-4ac3-a66e-10059191e5e5">

Right now there aren't security vulnerabilities, but I wanted something that would easily tell someone who's looking at the guide if/when there are, that way we don't have to come update the guide immediately